libprime_server.pc.in

```diff
@@ -1,7 +1,7 @@
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-libdir=@libdir@
-includedir=@includedir@
+prefix=@CMAKE_INSTALL_PREFIX@
```

Comment:
while this looks right for cmake, this will break for autotools which is the primary means by which i push things to the ppa. perhaps for cmake builds we an inline (heredoc in `CMakeLists.txt`) the `.pc` file?

Reply:
the problem is this project has two build systems, cmake and autotools. ive yet to figure out how to do ppa with cmake and so im hesitant to remove autotools completely until i do. i'm happy to make whatever changes to the cmake build system though that can accomodate your use case!